src/systems/systems/src/hydraulic/mod.rs

```diff
@@ -1622,6 +1625,185 @@ impl SimulationElement for LeakMeasurementValve {
     }
 }
 
+struct SpringPhysics {
+    k_constant: f64,
+    damping_constant: f64,
+
+    last_length: f64,
+}
+impl SpringPhysics {
+    fn new() -> Self {
+        Self {
+            k_constant: 5000.,
+            damping_constant: 500.,
+
+            last_length: 0.,
+        }
+    }
+
+    fn update_force_nm(
```

Comment:
What does the `nm` here stand for? Newton-meters? Newton and metres?

Reply:
Newton meters here, but as nothing here has true physical sense....  Not sure I should even mention units